### PR TITLE
[XRI3] Updating eye tracking sample scene so it doesn't use old XRI controllers

### PR DESCRIPTION
**Pull request by AMollis**
# Overview 

Updating an eye tracking sample scene so it doesn't use old XRI controllers

eye-tracking

### CODE REVIEW

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs [hunk 1]
```diff
@@ -33,62 +32,56 @@ public class FollowEyeGaze : MonoBehaviour
...
-        }
-        
-        private void OnDisable()
+        private void Update()
         {
-            if (_gazeTranslationAction == null || _gazeTranslationAction.action == null)
+            if (gazeInteractor == null)
             {
-                return;
+                return;
```
Comment (ms-RistoRK) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs:54]:
```suggestion
                return;
```

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs [hunk 2]
```diff
@@ -33,62 +32,56 @@ public class FollowEyeGaze : MonoBehaviour
         private Material material;
 
         [SerializeField]
-        private ActionBasedController gazeController;
+        [Tooltip("The TrackedPoseDriver that represents the gaze pose.")]
+        private TrackedPoseDriver gazePoseDriver;
```
Comment (ms-RistoRK) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs:36]:
```suggestion
        private TrackedPoseDriver gazeTrackedPoseDriver;
```

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs [hunk 3]
```diff
@@ -33,62 +32,56 @@ public class FollowEyeGaze : MonoBehaviour
...
 
-        private void FollowEyeGazeAction(InputAction.CallbackContext obj)
+        /// <summary>
+        /// Attempt to obtain the gaze transform.
+        /// </summary>
+        private bool TryGetGazeTransform(out Transform transform)
         {
-            // Example of obtaining gaze input action properties
-            Vector3 translation = _gazeTranslationAction.action.ReadValue<Vector3>();
+            if (gazePoseDriver != null)
```
Comment (ms-RistoRK) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs:75]:
```suggestion
            if (gazeTrackedPoseDriver != null)
```

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs [hunk 4]
```diff
@@ -33,62 +32,56 @@ public class FollowEyeGaze : MonoBehaviour
...
+            {
+                transform = gazePoseDriver.transform;
```
Comment (ms-RistoRK) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs:77]:
```suggestion
                transform = gazeTrackedPoseDriver.transform;
```

File: UnityProjects/MRTKDevTemplate/Assets/Scenes/EyeTracking/EyeTrackingBasicSetupExample.unity [hunk 5]
```diff
@@ -2031,6 +2031,14 @@ PrefabInstance:
       propertyPath: gazeController
       value: 
       objectReference: {fileID: 0}
+    - target: {fileID: 8364151238423769580, guid: c9d9edba85c2c504c962d7cd830d9e3e, type: 3}
+      propertyPath: gazeInteractor
+      value: 
+      objectReference: {fileID: 1778446323}
+    - target: {fileID: 8364151238423769580, guid: c9d9edba85c2c504c962d7cd830d9e3e, type: 3}
+      propertyPath: gazePoseDriver
```
Comment (ms-RistoRK) [inline UnityProjects/MRTKDevTemplate/Assets/Scenes/EyeTracking/EyeTrackingBasicSetupExample.unity:2039]:
```suggestion
      propertyPath: gazeTrackedPoseDriver
```

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs (hunk 2 above)
Comment (AMollis) [reply UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs:36]: Leaving this as is....

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs (hunk 4 above)
Comment (AMollis) [reply UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs:77]: keeping as is...the "Tracked" seems too wordy

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs (hunk 3 above)
Comment (AMollis) [reply UnityProjects/MRTKDevTemplate/Assets/Scripts/EyeTracking/FollowEyeGaze.cs:75]: Leaving as is

File: UnityProjects/MRTKDevTemplate/Assets/Scenes/EyeTracking/EyeTrackingBasicSetupExample.unity (hunk 5 above)
Comment (AMollis) [reply UnityProjects/MRTKDevTemplate/Assets/Scenes/EyeTracking/EyeTrackingBasicSetupExample.unity:2039]: Leaving as is

**Review by ms-RistoRK — APPROVED** [review]
Approved with suggestions.

**Review by shaynie — APPROVED** [review]
